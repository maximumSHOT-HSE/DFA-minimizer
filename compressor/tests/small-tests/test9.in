7 7 1
1 1 0
2 7 0
3 6 0
4 7 0
5 4 0
6 2 0
7 6 0
6
3
3 4 3 
