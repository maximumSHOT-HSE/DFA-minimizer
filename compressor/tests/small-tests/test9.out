4 36 9
2 1 1
4 1 1
2 1 2
4 1 2
1 1 6
3 1 6
1 1 8
3 1 8
1 2 0
2 2 0
3 2 0
4 2 0
3 2 2
3 2 3
2 2 5
3 2 5
4 2 5
2 2 6
4 2 6
2 2 7
4 2 7
1 3 1
3 3 1
1 3 2
1 3 3
2 3 3
4 3 3
1 3 4
2 3 4
3 3 4
4 3 4
1 3 5
1 3 7
3 3 7
2 3 8
4 3 8
2
1
2 
